Root/index.html

```diff
@@ -31,26 +47,25 @@
               </div>
             </td>
             <td class="dropdown">
-              <button class="dropbtn">My Sites</button>
-              <div class="content">
-                <a href="#">Northern Soul</a>
-                <a href="#">The Frying Pan</a>
-                <a href="Krzysztof/index.html" target="_blank">Krzysztof</a>
-              </div>
-            </td>
-            <td class="dropdown">
-              <button class="dropbtn">Work</button>
-              <div class="content">
-                <a href="#">R.H.Mawsons</a>
-                <a href="#">Stage One</a>
-                <a href="#">Wetherby Races</a>
+              <a href="#"><button class="dropbtn">Contact Me</button></a>
               </div>
             </td>
           </tr>
         </table>
       </div>
     </div>
+  <div class="main">
     <img src="images/picture2.jpg"/>
```

Comment:
The image is rather big for web size. Try resizing it in an editor to about 500px since you are not going to want it larger than that.